docs/js-quickstart.md

```diff
@@ -5,56 +5,55 @@ sidebar_label: Getting Started with JavaScript Testing
+    *   Add `.sauceignore `, `.sauce `and `.sauce/config.yml` for your framework using examples above, to your project
+    *   Update `config.yml` with test suite information including location and dependencies
 
- * Set your Sauce username and access key with `saucectl configure`
+ > For more detailed instructions please [consult the documentation](https://docs.saucelabs.com/testrunner-toolkit/installation), or see [the Cypress on Sauce Course](https://training.saucelabs.com/saucectl/)
 
- * Create a new SauceCTL project with `saucectl new`
 
- * Update `config.yml` with test suite information including location and dependencies
+## Step 2: Run Your Tests
 
- > For more detailed instructions please [consult the documentation](https://docs.saucelabs.com/testrunner-toolkit/installation), or see [an example with Cypress ](https://training.saucelabs.com/codelabs/Module1-Testrunner/index.html?index=..%2F..testrunner#2)
+By default your Cypress, TestCafe, or Playwright tests test suites are run on Sauce Labs VMs when you run the command:
 
+### `saucectl run`
+
+
+You have the option to run your tests in Docker mode or in a combination of _Docker mode_ and _Sauce mode_. To change these settings
+
+Set mode in `config.yml`:
+
+
+
+*   Set it as a global setting
+
+    ```
+    defaults:
+      mode: sauce
+    ```
+
+
+*   Set it on the suite level (override)
+
+    ```
+    Suites:
+     - name: saucy test suite
+       mode: docker
+
+    ```
 
-## Step 2: Run Your Tests
 
-You can run your Cypress, TestCafe, or Playwright tests in two different ways:
-
- * __Docker Mode__: Install Docker and run a containerized version of your test environment, then set mode in `config.yml`:
- 
- It can be set as a global setting.
- ```yaml
- defaults:
-   mode: docker
- ```
-
-It can also be set on suite level. The suite setting will override the global setting for the relevant suite.
- ```yaml
- suites:
-   - name: docker suite
-     mode: docker
- ```
- 
- * __Sauce VM Mode__: Install `saucectl` and pass your entire test suite, including dependencies and configurations, to Sauce Labs Cloud Virtual Machines. Update following in `config.yml` to run your tests on Sauce Labs VMs:
-
- It can be set as a global setting.
- ```yaml
- defaults:
-   mode: docker
- ```
-
-It can also be set on suite level. The suite setting will override the global setting for the relevant suite.
- ```yaml
- suites:
-   - name: docker suite
-     mode: docker
- ```
- 
-
- > To learn more about running tests please [consult the documentation](https://docs.saucelabs.com/testrunner-toolkit/running-tests), or see an [example with Cypress](https://training.saucelabs.com/codelabs/Module1-Testrunner/index.html?index=../..testrunner#3).
+> _Puppeteer is only supported in Docker Mode and will run in Docker mode automatically. See the [tutorial](https://training.staging.saucelabs.net/saucectl/) and [documentation](https://docs.saucelabs.com/testrunner-toolkit/running-tests) to learn more_
```

Comment:
Again- since this 'quickstart' applies to all Testrunner frameworks, I want to keep it in somewhere. Also having run these before, I still think Docker mode should be mentioned since it's useful for running quicker tests.